lib/ddtrace/contrib/http/circuit_breaker.rb

```diff
@@ -0,0 +1,39 @@
+
+module Datadog
+  module Contrib
+    module HTTP
+      # HTTP integration circuit breaker behavior
+      # For avoiding recursive traces.
+      module CircuitBreaker
```

Comment:
Might be a bit confusing with CircuitBreaker pattern. But I don't have better suggestion :D

Reply:
Is there a "circuit breaker" pattern? I was trying to think of something that encapsulated the idea of "if looping, escape", like a short-circuit where the circuit breaker trips to prevent an overload. But maybe a metaphor here isn't the best choice of naming scheme.

Reply:
FYI https://martinfowler.com/bliki/CircuitBreaker.html

on a high level it also breaks circuit, but its a bit more dynamic and serves a bit different purpose.

Reply:
Right... a microservices related pattern. Forgot about this one.